float32 right_velocity
float32 left_velocity
